bin/test.js

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env node
+import { ConfigLoaderError, readConfig } from '@web/config-loader';
+import commandLineArgs from 'command-line-args';
+import commandLineUsage from 'command-line-usage';
+import process from 'node:process';
+import { startTestRunner } from '@web/test-runner';
+import { WTRConfig } from '../src/server/wtr-config.js';
+
+const DISALLOWED_OPTIONS = ['--browsers', '--playwright', '--puppeteer', '--groups'];
+
+const optionDefinitions = [
+	// @web/test-runner options
+	{
+		name: 'files',
+		type: String,
+		multiple: true,
+		description: 'Test files to run. Path or glob.\n[Default: ./test/**/*.<type>.js]',
```

Comment:
```suggestion
		description: 'Test files to run. Path or glob.\n[Default: ./test/**/*.<group>.js]',
```

Reply:
Neither of these are _really_ accurate, right? For group `unit`, this is `test`.  For group `vdiff`, this is `vdiff`.

Reply:
Closing the loop. We decided the `unit` group was weird anyway, and it should just be a generic `test` group, which makes using `<group>` here accurate.